rust/src/scripts.rs

```diff
@@ -36,6 +36,10 @@ static IGNORED_PKG_SCRIPTS: phf::Set<&'static str> = phf_set! {
     "kernel-rt-modules.posttrans",
     "kernel-rt-debug-core.posttrans",
     "kernel-rt-debug-modules.posttrans",
+    "kernel-64k-core.posttrans",
+    "kernel-64k-modules.posttrans",
+    "kernel-64k-debug-core.posttrans",
+    "kernel-64k-debug-modules.posttrans",
```

Comment:
the `kernel-64k` packages don't really match the `kernel-rt` ones. 
Not sure if these are supposed to match the package names, but the packages I added in rhcos are slighly different. 
```suggestion
    "kernel-64k-modules-core.posttrans",
    "kernel-64k-modules-extra.posttrans",
```

Reply:
Hmm, I am not sure we need to handle -modules-core here...we probably *should* for consistency but:

```
[root@cosa-devsh ~]# rpm -qp --scripts kernel-64k-modules-core-5.14.0-362.el9.aarch64.rpm 
warning: kernel-64k-modules-core-5.14.0-362.el9.aarch64.rpm: Header V4 RSA/SHA256 Signature, key ID 8483c65d: NOKEY
postuninstall scriptlet (using /bin/sh):
rm -f /lib/modules/5.14.0-362.el9.aarch64+64k/modules.*
posttrans scriptlet (using /bin/sh):
/sbin/depmod -a 5.14.0-362.el9.aarch64+64k
[root@cosa-devsh ~]# 
```

That's mostly harmless.  It's the same for the other ones.

Reply:
cool ! I just wanted to be sure it was not overlooked :)